include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2905,13 +2905,13 @@ DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::
   Assert((this->dof_handler->hp_capability_enabled == false) ||
            (this->has_children() == false),
          ExcMessage(
-           "You can not ask for the future_fe_index on a cell that has "
+           "You can not ask for the future fe index on a cell that has "
            "children because no degrees of freedom are assigned "
            "to this cell and, consequently, no finite element "
            "is associated with it."));
   Assert((this->dof_handler->hp_capability_enabled == false) ||
            (this->is_locally_owned()),
-         ExcMessage("You can only query future_fe_index information on cells "
+         ExcMessage("You can only query future fe index information on cells "
```

Comment:
Here and elsewhere, I wonder whether if you already remove the underscore, we should improve readability by saying "(active|future) FE index" (with fe capitalized). This makes it clear that this part of the expression is supposed to be an abbreviation.